src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -14341,13 +14283,27 @@ static void Main()
 }
 ";
             var comp = CreateCompilation(source);
-            comp.VerifyDiagnostics(
-                // (6,9): error CS0102: The type 'C' already contains a definition for 'X'
+            comp.VerifyEmitDiagnostics(
+                // (4,10): error CS8652: The feature 'positional fields in records' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
```

Comment:
>// (4,10): error CS8652: The feature 'positional fields in records' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [](start = 16, length = 171)

Should we use explicit language version for this compilation so that the test doesn't break once default version is changed? It looks like this comment is applicable to other tests too.

Reply:
I checked a few such tests and they all use the `CreateCompilation` defined in this file with explicit language version 9.

---
In reply to: [612822814](https://github.com/dotnet/roslyn/pull/52480#discussion_r612822814) [](ancestors = 612822814)